setup.py

```diff
@@ -18,15 +18,38 @@
 
 from setuptools import setup
 
+
+def assert_pyspark_version():
```

Comment:
@AbdealiJK this logic looks good to me, but I would put this function instead in this file: 

https://github.com/databricks/spark-pandas/blob/master/databricks/koala/__init__.py#L22

I do not think it would work as expected here since this file is the setup file.

Reply:
I did not keep it on Line 22 - as if we're using an import that is not valid in an older pyspark, it would throw an ImportError and the warning would not be shown.
Added it to the top.